README.md

```diff
@@ -1,404 +1,3 @@
-      maxSurge: 1
-      maxUnavailable: 1
-    type: RollingUpdate
-  template:
-    metadata:
-      labels:
-        app: npm-proxy
-    spec:
-      containers:
-        - image: sourcegraph/npm-proxy:latest
-          name: npm-proxy
-          ports:
-            - containerPort: 8080
-              name: http
-          resources:
-            limits:
-              cpu: '1'
-              memory: 1Gi
-          volumeMounts:
-            - mountPath: /cache
-              name: npm-proxy-cache
-      volumes:
-        - name: npm-proxy-cache
-          persistentVolumeClaim:
-            claimName: npm-proxy
----
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  annotations:
-    volume.beta.kubernetes.io/storage-class: default
-  name: npm-proxy
-spec:
-  accessModes:
-    - ReadWriteOnce
-  resources:
-    requests:
-      storage: 100Gi
----
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: npm-proxy
-  name: npm-proxy
-spec:
-  ports:
-    - name: http
-      port: 8080
-      targetPort: http
-  selector:
-    app: npm-proxy
-  type: ClusterIP
-```
-
-Then define a `.yarnrc` as a config map that points to the proxy:
-
-```yaml
-apiVersion: v1
-data:
-  .yarnrc: |
-    # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-    # yarn lockfile v1
-
-
-    https-proxy "http://npm-proxy:8080"
-    proxy "http://npm-proxy:8080"
-    strict-ssl false
-kind: ConfigMap
-metadata:
-  name: yarn-config
-```
-
-and mount it into the container:
-
-```diff
-  name: lang-typescript
-+ volumeMounts:
-+  - mountPath: /yarn-config
-+    name: yarn-config
-```
-
-```diff
-  spec:
-+   volumes:
-+     - configMap:
-+         name: yarn-config
-+       name: yarn-config
-```
-
-## Configuration
-
-### Support for dependencies on private packages and git repositories
-
-Dependencies on private npm packages and private registries is supported by setting the `typescript.npmrc` setting.
-It contains the same key/value settings as your `.npmrc` file in your home folder, and therefor supports the same scoping to registries and package scopes.
-See https://docs.npmjs.com/misc/config#config-settings for more information on what is possible to configure in `.npmrc`.
-
-Example:
-
-```json
-"typescript.npmrc": {
-  "//registry.npmjs.org/:_authToken": "asfdh21e-1234-asdn-123v-1234asdb2"
-}
-```
-
-For dependencies on private git repositories, mount an SSH key into `~/.ssh`.
-
-## Contributing
-
-You need NodeJS >=11.1.0 and yarn installed.
-
-```sh
-# Install dependencies
-yarn
-# Build the extension and the server
-yarn run build
-```
-
-To debug the server, build, then run it locally with `yarn start-server` and point the extension to it through Sourcegraph settings:
-
-```json
-"typescript.serverUrl": "ws://localhost:8080"
-```
-
-To debug the extension, serve the extension from localhost with `yarn serve-ext` and [sideload it into Sourcegraph](https://docs.sourcegraph.com/extensions/authoring/local_development).
+TODO
```

Comment:
We should probably at least put a link here, before merging.

Reply:
Oops, good catch.

Reply:
I'm not linking to the extension as they're unrelated, but I did put in a quick blurb about the project.